src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
```

Comment:
The `link` tag should follow the same attribute formatting rule as `meta` tags. Please format the attributes so that each starts on a new line with the correct indentation.